src/Magnum/ImGuiIntegration/Widgets.h

```diff
@@ -72,18 +72,40 @@ inline ImTextureID textureId(GL::Texture2D& texture) {
 @param size         Widget size
 @param uvRange      UV range on the texture (covers the whole texture by
     default)
-@param tintColor    Tint color, default @cpp 0xffffffff_rgbaf @ce
-@param borderColor  Border color, default @cpp 0x00000000_rgbaf @ce
 
 @see @ref textureId()
 */
 inline void image(GL::Texture2D& texture, const Vector2& size,
-    const Range2D& uvRange = {{}, Vector2{1.0f}},
-    const Color4& tintColor = Color4{1.0f},
+    const Range2D& uvRange = {{}, Vector2{1.0f}})
+{
+    ImGui::Image(textureId(texture), ImVec2(size), ImVec2(uvRange.topLeft()), ImVec2(uvRange.bottomRight()));
+}
+
+#ifdef MAGNUM_BUILD_DEPRECATED
+/**
+@brief Image widget displaying a @ref GL::Texture2D
+@m_deprecated_since_latest Using explicit tint and border color is no longer
+    possible in newer ImGui. Use @ref image(GL::Texture2D&, const Vector2&, const Range2D&)
+    instead.
```

Comment:
Can you mention this in `doc/changelog-integration.dox`, potentially updating other deprecation notices for `image()`?

And isn't the same case with tint / border in `imageButton()`?

Reply:
Added. `ImGui::ImageButton()` still has the explicit tint and border color. I don't know why and I don't really want to know, either 🐱

Reply:
To be honest, I think all of these widget wrappers should be deprecated, seeing how much pain they've already caused. And with `textureId()` now being a thing, there's really no good reason to use these.